docs/releases/updating_ez_platform.md

```diff
@@ -3,53 +3,106 @@
 
 This page explains how to update eZ Platform to a new version.
 
-In the instructions below, replace` <version>` with the version of eZ Platform you are updating to (for example: `v1.7.0`). If you are testing a release candidate, use the [latest rc tag](https://github.com/ezsystems/ezplatform/releases) (for example: `v1.7.1-rc1`).
+In the instructions below, replace `<version>` with the version of eZ Platform you are updating to (for example: `v1.7.0`). If you are testing a release candidate, use the latest rc tag (for example: `v1.7.1-rc1`).
 
 ## Update procedure
 
 ## 1. Check out a tagged version
 
-**1.1.** From the project's root, create a new branch from the project's master, or from the branch you're updating on:
+**1.1.** From the project's root, create a new branch from the project's "master", or from the branch you're updating on:
 
-**From your master branch**
+**From your master branch, create a branch for handling update changes**
 
 ``` bash
 git checkout -b <branch_name>
 ```
 
 This creates a new project branch for the update based on your current project branch, typically `master`. An example `<branch_name>` would be `update-1.4`.
 
-**1.2.** If it's not there, add `ezsystems/ezplatform` (or `ezsystems/ezplatform-ee`, when updating an Enterprise installation) as an upstream remote:
+**1.2.** If it's not there, add `ezsystems/ezplatform` as an "upstream" remote
+(on an Enterprise installation use `ezsystems/ezplatform-ee`, and on an eZ Commerce installation, `ezsystems/ezcommerce`):
 
 **From your new update branch**
 
-``` bash
-git remote add ezplatform http://github.com/ezsystems/ezplatform.git
+``` bash hl_lines="1 3 5"
+git remote add upstream http://github.com/ezsystems/ezplatform.git
+or
+git remote add upstream http://github.com/ezsystems/ezplatform-ee.git
 or
-git remote add ezplatform-ee http://github.com/ezsystems/ezplatform-ee.git
+git remote add upstream http://github.com/ezsystems/ezcommerce.git
 ```
 
-**1.3.** Then pull the tag into your branch.
+**1.3** Prepare for pulling changes
+
+??? note "Adding `sort-packages` option when updating from <1.7.9, 1.13.5, 2.2.4, 2.3.3, 2.4.0"
```

Comment:
Could we change the way the versions are listed here? Right now it's a bit confusing, because there is no 2.3.3, 2.4.0 yet (and the difference between `<` and `<=` can be easily overlooked).
Would it be OK to change it to: `updating from <=1.7.8, 1.13.4, 2.2.3, 2.3.2`?

Reply:
yes, that is fine with me